source/guides/guides/continuous-integration.md

```diff
@@ -723,6 +723,21 @@ module.exports = (on, config) => {
 }
 ```
 
+In addition, {% issue 2102 "this issue" %} reports that headless runs do not render a full-size image for viewports larger than 1280x720. To test larger viewports, set `viewportWidth` and `viewportHeight` in `cypress.json`, and amend the shared memory fix to set the maximum window size:
```

Comment:
The `viewportWidth` and `viewportHeight` have nothing to do with the window size. This is the size that the application will render at *within* the Cypress Test Runner. We create an iframe and show the preview at these dimensions.